lib/Cleantalk/Common/API.php

```diff
@@ -23,6 +23,7 @@ class API
     const URL = 'https://api.cleantalk.org';
     const AGENT = 'ct-api-3.2';
 
+    protected static $url;
```

Comment:
It seems no matters to be added this property.

Reply:
Fixed in new commits